src/docs/patterns/user-auth.mdx

```diff
@@ -0,0 +1,324 @@
+
+### Screens
+
+The majority of the joining-an-organization screens are the same regardless of the underlying mechanism. Use the following screens for the joining-an-organization workflow where appropriate.
+
+<ImageGrid
+    images={[EnterOrganizationCode, EnterOrganizationPasscode, EnterOrganizationConfirm, EnterOrganizationWelcome]}
+    caption={[
+        'Screens in the joining-an-organization workflow.',
+        '1: Enter organization identifier',
+        '2: Enter organization passcode',
+        '3: Success: request sent',
+        '4: Success: organization joined',
+    ]}
+/>
+<ImageGrid images={[JoinAnOrganizationDesktop]} caption={'A join-an-organization screen example on desktop.'} regularWidth />
+
+## Login
+
+<!--
+An NFC login might either be documented here, or wait until we start adding in device registery / discovery pattern
+-->
+
+Logging in can usually be accomplished in one of two ways: basic login or login via a third-party.
+
+**Basic Login**
+
+A simple login presents fields for the users to enter their credentials and a button to submit them for verification.
+
+The login screen should always offer some workaround if a user has some trouble getting authorized, either because they don't have an account, or that they forget their credentials. In your login screen, there should be buttons that can initiate those extra functions, such as Registration and Forgot Password (see below).
+
+<ImageGrid
+    images={[Login]}
+    caption={[
+        'A standard login screen.',
+        '1: Product identifications, such as product name or logo.',
+        '2: Optionally, you may add in a simple geometrical background image with a white gradient background.',
+        '3: "Remember me" and "password visibility" are two features that facilitate the user, but expose some security concerns.',
+        '4: Login screens often include extra functions, such as "forgot password", "sign up", and "contact your organization admin".',
+        '5: You may add certification badges and logos from partnerships here.',
+    ]}
+    gridImageProps={{ sm: 8 }}
+/>
+<ImageGrid images={[LoginDesktop]} caption={'A login screen example on desktop.'} regularWidth />
+
+**Login via Third Party**
+
+Some applications, especially consumer-facing ones, can be authorized via third-party accounts / credentials. Different platforms may have different requirements for this. Here are some commonly used platforms and their instructions:
+
+-   [Login with Amazon](https://developer.amazon.com/docs/login-with-amazon/brand.html)
+-   [Google Sign-in](https://developers.google.com/identity/sign-in/web/sign-in)
+-   [Sign in with Apple](https://developer.apple.com/design/human-interface-guidelines/sign-in-with-apple/overview/introduction/)
+-   [Facebook Login](http://developers.facebook.com/docs/facebook-login/)
+-   [Sign in with Microsoft](https://developer.microsoft.com/en-us/identity/add-sign-in-with-microsoft)
+
+## Forgot / Reset Password
```

Comment:
```suggestion
## Forgot / Change Password
```

Reply:
Does "reset password" include both forget password and change password?

Reply:
I think Forgot covers Forgot/Reset Password and
Change Password covers change password.

Forgot/Reset is slightly different from Change because Change happens when you are already authenticated whereas Reset has an extra step of sending the email.